examples/pytorch/requirements.txt

```diff
@@ -1,3 +1,4 @@
-torch-geometric
-torch-scatter
-torch-sparse
+horovod[torch]==0.24.3
```

Comment:
We should be able to remove this file since these requirements are already in the main one?

Reply:
This is needed for E2E testing without bazel: pip install wheels on a clean instance and run scripts. I added horovod and versions here [for caching](https://github.com/actions/setup-python#caching-packages-dependencies).